torch/_ops.py

```diff
@@ -25,6 +24,79 @@ def dl_open_guard():
     if _SET_GLOBAL_FLAGS:
         sys.setdlopenflags(old_flags)
 
+class OpOverload:
+    def __init__(self, op, schema):
+        self.op = op
+        self.schema = schema
+
+    def __deepcopy__(self, memo=None):
+        return self
+
+    def __call__(self, *args, **kwargs):
+        return self.op(*args, **kwargs or {})
+
+    def __getattr__(self, key):
+        return getattr(self.op, key)
+
+    # `my_namespace::my_op`
+    def qualified_name(self):
+        return self.schema.name
+
+    def overload_name(self):
+        return self.schema.overload_name
+
+    # vector of Arguments -- each Argument's name, and type can currently be accessed
+    # it might be useful to make the mutability of the arg accessible (using the alias_info stored in the C++ struct)
+    def returns(self):
+        return self.schema.returns
+
+    def arguments(self):
+        return self.schema.arguments
+
+class OpOverloadPacket():
+    def __init__(self, qualified_op_name, op_name, op):
+        self.qualified_op_name = qualified_op_name
+        self.op_name = op_name
+        self.op = op
+
+    def __deepcopy__(self, memo=None):
+        return self
+
+    # def __repr__(self):
```

Comment:
nit: debug print.
Also if we want to add one here, it should be `__str__` I think?

Reply:
yup added it